components/eamxx/src/physics/nudging/eamxx_nudging_process_interface.cpp

```diff
@@ -523,8 +523,8 @@ void Nudging::run_impl (const double dt)
         apply_weighted_tendency(atm_state_field, int_state_field, nudging_weights_field, dt);
       } else if (m_refine_remap_vert_cutoff > 0.0) {
```

Comment:
Would it make more sense to set the default `cutoff` value to something huge rather than 0.0?  In it's current implementation everything works because the `if` conditional checks if `cutoff>0.0` to begin with, but if someone messes that up the code would run fine and cutoff all nudging (so do nothing).  Maybe that's an argument for keeping it, so that if someone disrupts the code the baseline is not do any nudging at all...  I welcome push back.

Reply:
One hand, I agree with Aaron, in the sense that not doing cutoff is the same as having the cutoff set to something very large (larger than any possible surface pressure), so having it set to 200k should be a good safeguard. However, that means you should change the check in this line (you can't check the cutoff itself), and you to check something else.

All in all, I think the current impl is fine.

Side comment though: @mahf708 I noticed going through the file that there are some indented blocks that use tabs instead of spaces. That can be messy to visualize, since different editors may use different tab length. If you have a second, can you sed `\t` with two spaces (which seems to be the indentation across the file)?

(btw: the `\t`->`  ` replacement thing is one of those things we ought to automate via actions...)

Reply:
E.g., starting at line 417, I see this
```c++
      for (int kk=fill_idx+1; kk<num_src_levs; ++kk) {
        const auto ipack = kk / mPack::n;
  const auto iidx  = kk % mPack::n;
        // Check if this index is masked
  if (!(ext_state_view_1d(ipack)[iidx]==var_fill_value)) {
    fill_value = ext_state_view_1d(ipack)[iidx];
  } else {
    ext_state_view_1d(ipack)[iidx] = fill_value;
  }
      }   
    }); 
```
It is all outside your mods, I was just hoping that we could piggy back that mod in... No big deal though.

Reply:
Ok, I thought this was a 5sec edit, but if it's raising other issues, forget about it. Feel free to force push to prev version...

Reply:
It still displays wrong on my editor, so I'd say reset HEAD~1 and use the force ...

Reply:
We actually _do_ have a `.clang_format` file in the eamxx folder. It's been there since the start of scream, but we never used it. Maybe we can revive it, and add a git hook (or gh action) to apply formatting when PR are merged...